ceph.spec.in

```diff
@@ -99,7 +99,7 @@
 %endif
 %bcond_with seastar
 %bcond_with jaeger
-%if 0%{?fedora} || 0%{?suse_version} >= 1500
+%if 0%{?fedora} || 0%{?openEuler} || 0%{?suse_version} >= 1500
```

Comment:
This is a snippet for `cephfs_shell`.  In main, it's not conditioned on `openEuler` in any way:
```
%if 0%{?fedora} || 0%{?suse_version} >= 1500 || 0%{?rhel} >= 9
# distros that ship cmd2 and/or colorama
%bcond_without cephfs_shell
%else
# distros that do _not_ ship cmd2/colorama
%bcond_with cephfs_shell
%endif
```
Why is it getting conditioned on openEuler here?

Reply:
Sorry, wrong pick. Fixed

Reply:
The fixed version looks good to me, but such conflicts should be mentioned in the commit message.  See https://github.com/ceph/ceph/blob/main/SubmittingPatches-backports.rst#cherry-picking-rules:
> * the commit message generated by ``git cherry-pick -x`` must not be modified, except to add a "Conflicts" section below the "cherry picked from commit ..." line added by git
> * the "Conflicts" section must mention all files where changes had to be made manually (not just conflicts flagged by git)
> * the "Conflicts" section should also describe the manual changes that were made




At a minimum, the lines generated by git should have been uncommented:
```
(cherry picked from commit 14c1f3ddd627d752563a268a534870a921256996)

Conflicts:
        ceph.spec.in
```
But it's always nicer to note the cause of the conflict, and also the steps that were taken to resolve if the resolution is complex:
```
(cherry picked from commit 14c1f3ddd627d752563a268a534870a921256996)

Conflicts:
        ceph.spec.in [ commit 7be8be63501b ("ceph.spec.in: make jaeger
          packages installation default true.") not in quincy ]
```

Reply:
Learned a course. Fixed and thanks~